backend/dal/Migrations/v01.07.02/Up/PreUp/01-Buildings.sql

```diff
@@ -0,0 +1,423 @@
+PRINT 'Transfer Capital Planning Branch Buildings to School Districts'
```

Comment:
Identified all `AdministrativeAreas` within our DB and migrated them to an appropriate school district.  This took forever...